services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    with transaction.atomic():
+        order = Order.objects.create(user=User.objects.get(username=username))
+        if date:
+            order.created_at = date
+            order.save()

```

Comment:
You are calling `save()` method two times if the `date` is not `None`. According to the checklist, you should call `save()` method only one time if it's possible. You can create `Order` instance with `created_at` field if `date` is not `None`.